src/spetlr/delta/delta_handle.py

```diff
@@ -207,3 +237,19 @@ def delete_data(
             f" WHERE {comparison_col} {comparison_operator} {limit};"
         )
         Spark.get().sql(sql_str)
+
+    def read_stream(self) -> DataFrame:
+        if Spark.version() < Spark.DATABRICKS_RUNTIME_10_4:
+            raise SparkVersionNotSupportedForSpetlrStreaming()
```

Comment:
please put this into the init method. I don't want to wait until the orchestrator executes to get this error.

Reply:
@mrmasterplan  - in my oppinion, it should not be in the init method. The deltahandle can be usd for other stuff than streaming.

Reply:
@JeppeBlixen had a good point - I will remove this raise:

*"When having a spark version check here, is it necessary on the readStream() handlers?
I would argue it is needed here because we use the availableNow, but someone could be using the handler.readStream() with there own loader?"*